### PR TITLE
feat: [AXIMST-739] added xblock actions

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
[YouTrack](https://youtrack.raccoongang.com/issue/AXIMST-739/Add-actions-for-tree-dots-hblocks)

### CODE REVIEW

File: src/course-unit/__mocks__/courseSectionVertical.js [hunk 1]
```diff
@@ -318,10 +318,12 @@ module.exports = {
     ],
     has_changes: true,
     actions: {
```
Comment (ruzniaievdm) [inline src/course-unit/__mocks__/courseSectionVertical.js:320]: Please leave everything as it was. The data in this block does not apply to our changes

File: src/course-unit/course-xblock/CourseXBlock.jsx [hunk 2]
```diff
@@ -136,28 +139,36 @@ const CourseXBlock = ({
...
-                  <Dropdown.Item>
-                    {intl.formatMessage(messages.blockLabelButtonMove)}
-                  </Dropdown.Item>
-                  {canEdit && (
+                  {canMove && (
+                    <Dropdown.Item>
+                      {intl.formatMessage(messages.blockLabelButtonMove)}
+                    </Dropdown.Item>
+                  )}
+                  {canEdit && canCopy && (
```
Comment (ruzniaievdm) [inline src/course-unit/course-xblock/CourseXBlock.jsx:157]: [question] Why did you add `canEdit` here too? What does this selector process?

File: src/course-unit/course-xblock/CourseXBlock.jsx [hunk 3]
```diff
@@ -136,28 +139,36 @@ const CourseXBlock = ({
...
-                  <Dropdown.Item onClick={() => unitXBlockActions.handleDuplicate(id)}>
-                    {intl.formatMessage(messages.blockLabelButtonDuplicate)}
-                  </Dropdown.Item>
-                  {getConfig().ENABLE_TAGGING_TAXONOMY_PAGES && (
+                  {canDuplicate && (
+                    <Dropdown.Item onClick={() => unitXBlockActions.handleDuplicate(id)}>
+                      {intl.formatMessage(messages.blockLabelButtonDuplicate)}
+                    </Dropdown.Item>
+                  )}
+                  {canManageTags && getConfig().ENABLE_TAGGING_TAXONOMY_PAGES && (
```
Comment (ruzniaievdm) [inline src/course-unit/course-xblock/CourseXBlock.jsx:147]: [optional] Swap these variables

File: src/course-unit/course-xblock/CourseXBlock.jsx (hunk 2 above)
Comment (PKulkoRaccoonGang) [reply src/course-unit/course-xblock/CourseXBlock.jsx:157]: Earlier @monteri  said that I should add `canEdit` for this button and other copy/paste functionality
@monteri what do you think?

File: src/course-unit/course-xblock/CourseXBlock.jsx (hunk 3 above)
Comment (PKulkoRaccoonGang) [reply src/course-unit/course-xblock/CourseXBlock.jsx:147]: Swapped

File: src/course-unit/course-xblock/CourseXBlock.jsx (hunk 2 above)
Comment (PKulkoRaccoonGang) [reply src/course-unit/course-xblock/CourseXBlock.jsx:157]: At the stand up we decided that we would get rid of the `canEdit`

File: src/course-unit/__mocks__/courseSectionVertical.js (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply src/course-unit/__mocks__/courseSectionVertical.js]: Removed, thanks

**Review by monteri — APPROVED** [review]
